### PR TITLE
Additional updates for the HPVS module

### PR DESCRIPTION
**Pull request by mloviska**
Accidentaly I have merged the previous PR prematurely. There is an account change that has different settings and we need to pull s390x image specifically as the platform runs on this architecture.

- ticket: https://progress.opensuse.org/issues/163190
- Verification run: http://kepler.suse.cz/tests/23796#step/prepare_hpvs/69

### CODE REVIEW

**Review by ricardobranco777 — APPROVED** [review]

**Comment by m-dati** [comment]
I was checking too, but ok :smile: Merged. LGTM

File: tests/containers/prepare_hpvs.pm [hunk 1]
```diff
@@ -23,7 +23,7 @@ my $ibm = {
     api => get_var('IBM_API'),
     region => 'eu-gb',
     ingestionKey => get_var('LOG_KEY'),
-    instance => 'testing-openqa',
+    instance => 'openqa-',
```
Comment (m-dati) [inline tests/containers/prepare_hpvs.pm:26]: Only kind question: what is the rule behind this lexeme change?

File: tests/containers/prepare_hpvs.pm [hunk 2]
```diff
@@ -100,6 +100,7 @@ sub find_entity {
 
 sub run {
     select_serial_terminal();
+    $ibm->{instance} .= lc(sprintf("%08X", rand(0xFFFFFFFF)));
```
Comment (m-dati) [inline tests/containers/prepare_hpvs.pm:103]: Are you creating a new uniq identifier here?

Comment (mloviska) [reply tests/containers/prepare_hpvs.pm:103]: yes, we might need to run containers of multiple versions

File: tests/containers/prepare_hpvs.pm (hunk 1 above)
Comment (m-dati) [reply tests/containers/prepare_hpvs.pm:26]: Ah I see continued in L103

**Pull request merged**